plugin-server/src/cdp/utils/hog-function-filtering.ts

```diff
@@ -321,6 +351,29 @@ export async function filterFunctionInstrumented(options: {
     }
 
     try {
+        // If there are no filters, everything matches no need to execute bytecode (lets save those cpu cycles)
+        if (!filters?.actions && !filters?.events && !filters?.filter_test_accounts) {
+            result.match = true
+            return result
+        }
+        // Quick pre-filter check to avoid running bytecode for obvious non-matches
+        if (shouldBePreFilteredBasedOnEventName(filters, filterGlobals)) {
```

Comment:
I like this but im not super certain of the logic. could we first just metric it.

So we save the fact that we would have prefiltered it out, then we compare to whether it was dropped. If there is any case where we _would_ have pre-filtered but _didnt_ drop then we would want to metric that outcome as it means we have an error